usr/share/rear/restore/BORG/default/30_load_archives.sh

```diff
@@ -0,0 +1,64 @@
+# This file is part of Relax-and-Recover, licensed under the GNU General
+# Public License. Refer to the included COPYING for full text of license.
+#
+# 10_load_archives.sh
+
+LogPrint "Starting Borg restore"
+
+# Do we have Borg binary?
+# Missing Borg binary should not happen as it is copyied by
+# backup/BORG/default/20_start_backup.sh
+has_binary borg
+BugIfError "Could not find Borg binary"
+
+# Query Borg server for repoitory information and store it to archive_cache.
+# This should avoid repeatingly quering Borg server, which could be slow.
+archive_cache=$TMP_DIR/borg_archive
+borg list $BORG_USERNAME@$BORG_HOST:$BORG_REPO > $archive_cache
+StopIfError "Could not list Borg archive"
+
+# Store number of lines in archive_cache file for later use
+archive_cache_lines=$(wc -l $archive_cache | awk '{print $1}')
+
+# This should not happen.
+# In theory if something goes wrong with obtaining Borg archive info,
+# we should be stopped by foregoing "StopIfError()"
+if [ archive_cache_lines -eq 0 ]; then
+    BugIfError "Something went wrong while getting Borg repository content"
+fi
+
+# Display list of archives in repository
+# Display header
+echo ""
+echo "=== Borg archives list ==="
+echo "Host:       $BORG_HOST"
+echo "Repository: $BORG_REPO"
+echo ""
+
+# Display archive_cache file content and prompt user for archive to restore
+while(true); do
```

Comment:
You are a Borg user and you implemented it,
so that your point of view matters most and
"always ask" is perfectly o.k. for me - in particular
now after you explained why it is this way (the latter
is a hint about a possibly missing explanatory comment
it the code so that others understand the reason behind ;-)